tests/lib/acorn-after-espree.js

```diff
@@ -21,7 +23,6 @@ describe("acorn", () => {
     it("acorn.parse() should work after espree was loaded.", () => {
         const before = acorn.parse("var foo = bar /*world*/;");
 
-        require("../../espree");
```

Comment:
I think this test doesn't make sense if we load `espree` upfront.

Reply:
should be fixed via https://github.com/eslint/espree/pull/469/commits/9e82484031a93f606ed12b9fb6f6d42535093f6c